src/libraries/Directory.Build.props

```diff
@@ -124,10 +124,11 @@
 
     <!-- There are no WebAssembly tools, so use the default ones -->
     <_buildingInOSX>$([MSBuild]::IsOSPlatform('OSX'))</_buildingInOSX>
-    <ToolRuntimeRID Condition="'$(RuntimeOS)' == 'browser' and '$(TargetOS)' == 'windows'">win-x64</ToolRuntimeRID>
+    <_buildingInWindows>$([MSBuild]::IsOSPlatform('windows'))</_buildingInWindows>
+    <ToolRuntimeRID Condition="'$(RuntimeOS)' == 'browser' and $(_buildingInWindows)">win-x64</ToolRuntimeRID>
     <ToolRuntimeRID Condition="'$(RuntimeOS)' == 'browser' and '$(TargetOS)' != 'windows' and $(_buildingInOSX)">osx-x64</ToolRuntimeRID>
-    <ToolRuntimeRID Condition="'$(RuntimeOS)' == 'browser' and '$(TargetOS)' != 'windows' and !$(_buildingInOSX)">linux-x64</ToolRuntimeRID>
-
+<!-- this line seems to cause a failure with _SuppressSdkImports .  Doesn't make sense to me -->
+<!--    <ToolRuntimeRID Condition="'$(RuntimeOS)' == 'browser' and '$(TargetOS)' != 'windows' and !$(_buildingInOSX)" and !$(_buildingInWindows)">linux-x64</ToolRuntimeRID> -->
```

Comment:
I don't understand this.  The condition shouldn't even be met as `buildingInWindows` is true so why it causes an error I don't know.

Reply:
I guess it may a problem with evaluation order.

Reply:
Merging the latest got rid of this